src/hooks/useFocus.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useRef } from 'react';
+
+export const useFocus = (length: number) => {
+  const refs = Array.from({ length }, () => useRef<HTMLInputElement>(null));
+
+  //TODO: 다음 input으로 이동
+
+  useEffect(() => {
+    if (refs[0].current) {
+      refs[0].current.focus();
+    }
+  }, []);
```

Comment:
함수화를 똑같이 신경써보아요. 코드를 이해하는데 시간이 조금 소요되었어요 ㅎㅎ

이렇게 네이밍 붙여서, useEffect 를 쓸 수도 있어요. 

```suggestion
  useEffect(function focusOnFirstRef() {
        if (refs[0].current) {
            refs[0].current.focus();
        }
    }, []);
```

Reply:
수정했습니다!

36bd348eaea9dabd1d76e0e9f968eb4f7a3fea0d